plugins/annotation/__init__.py

```diff
@@ -0,0 +1,1125 @@
+"""
+Annotation operators.
+
+| Copyright 2017-2023, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+import json
+
+from bson import json_util
+
+import fiftyone as fo
+import fiftyone.operators as foo
+import fiftyone.operators.types as types
+import fiftyone.utils.annotations as foua
+
+
+class RequestAnnotations(foo.Operator):
+    @property
+    def config(self):
+        return foo.OperatorConfig(
+            name="request_annotations",
+            label="Request annotations",
+            dynamic=True,
+        )
+
+    def resolve_input(self, ctx):
+        inputs = types.Object()
+
+        request_annotations(ctx, inputs)
+
+        view = types.View(label="Request annotations")
+        return types.Property(inputs, view=view)
+
+    def execute(self, ctx):
+        kwargs = ctx.params.copy()
+        target = kwargs.pop("target", None)
+        anno_key = kwargs.pop("anno_key")
+        backend = kwargs.pop("backend")
+
+        # Parse label schema
+        kwargs.pop("schema_type")
+        label_schema = kwargs.pop("label_schema", None)
+        label_schema_fields = kwargs.pop("label_schema_fields", None)
+        if label_schema:
+            label_schema = json.loads(label_schema)
+        elif label_schema_fields:
+            label_schema = _build_label_schema(label_schema_fields)
+
+        # Parse backend-specific parameters
+        _get_backend(backend).parse_parameters(ctx, kwargs)
```

Comment:
I'm not sure what's happening here. Should the return of `_get_backend` be stored as a variable and used downstream?